.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-         python-version: ['3.8', '3.12']
+         python-version: ['3.7', '3.12']
```

Comment:
Actually, I suggest you bump this to 3.9, 3.12.

Reply:
The tutor v15.0.0 supports Python 3.7. I want everyone to be able to use this.